proto/src/proto.rs

```diff
@@ -418,6 +419,33 @@ pub struct LdapMatchingRuleAssertion {
     pub dn_attributes: bool, // DEFAULT FALSE
 }
 
+impl LdapMatchingRuleAssertion {
+    pub fn from_strings(left: String, right: String) -> Self {
+        let match_value = right.to_string();
```

Comment:
Nitpick: allocation is not required
```suggestion
        let match_value = right;
```